addons/web/static/src/webclient/actions/client_actions.js

```diff
@@ -14,6 +14,7 @@ export const displayNotificationAction = (env, action) => {
         sticky: params.sticky || false,
         title: params.title ? escape(params.title) : "",
         type: params.type || "info",
+        messageIsHtml: params.messageIsHtml || false,
```

Comment:
Actually, the `params.message` is _not_ html here, it's the `params.links` that must be preserved. The `params.message` content is currently always escaped. If `messageIsHtml` is `false`, the generated content will be escaped twice. The title is always escaped twice.

Reply:
`params.message` is never HTML (see below it's always escaped), so there are two possibilities:

* either keep the code as-is and always set messageIsHtml here
* or only escape the message and set `messageIsHtml` if links are found